share/logger.py

```diff
@@ -3,13 +3,16 @@
 # you may not use this file except in compliance with the Elastic License 2.0.
 
 import logging
+import os
 
 import ecs_logging
 from elasticapm.handlers.logging import LoggingFilter
 
+log_level = logging.getLevelName(os.getenv("LOG_LEVEL", logging.INFO))
```

Comment:
log level is `INFO` by default: it can be overridden by `LOG_LEVEL` env variable